From 25beba1efc31901a3bb0b1b6f0604d6583dc0513 Mon Sep 17 00:00:00 2001
From: Alexander Kanavin <alex.kanavin@gmail.com>
Date: Fri, 20 Jan 2017 13:32:06 +0200
Subject: [PATCH] Do not reset the PATH environment variable before running
 scriptlets.

We add lots of native stuff into it and scriptlets rely on that.

Upstream-Status: Inappropriate [oe-core specific]
Signed-off-by: Alexander Kanavin <alex.kanavin@gmail.com>

---
 lib/rpmscript.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/rpmscript.c b/lib/rpmscript.c
index 36e37cf77..37ada014c 100644
--- a/lib/rpmscript.c
+++ b/lib/rpmscript.c
@@ -252,7 +252,7 @@ static void doScriptExec(ARGV_const_t argv, ARGV_const_t prefixes,
 	if (ipath && ipath[5] != '%')
 	    path = ipath;
 
-	xx = setenv("PATH", path, 1);
+	//xx = setenv("PATH", path, 1);
 	free(ipath);
     }
 
